File: MODUL 4/SOAL 5/PRAK405-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);
    int total = 0;
    for (int i = 1; i <= n; i++) {
        int baris_total = 0;
        printf("(");
        for (int j = i; j >= 1; j--) {
            int hasil = j * kelipatan;
            baris_total += hasil;
            printf("%d * %d", j, kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        } printf(") = %d\n", baris_total);
        total += baris_total;
    } printf("%d\n", total);
    return 0;
}

File: MODUL 4/SOAL 5/PRAK405-2410817210002-Sebastian_Abe_Santoso.py
a = []
while len(a) != 2:
    b = input()
    a.extend(map(int, b.split()))
n, kelipatan = a[:2]
total = 0
for i in range(1, n + 1):
        baris_total = 0
        print("(", end="")
        for j in range(i, 0, -1):
            hasil = j * kelipatan
            baris_total += hasil
            print(f"{j} * {kelipatan}", end="")
        if j > 1:
            print(" + ", end="")
        print(f") = {baris_total}")
        total += baris_total
print(total)